src/reason_lexer.mll

```diff
@@ -510,6 +517,7 @@ rule token = parse
   | "-"  { MINUS }
   | "-." { MINUSDOT }
   | "<>" { LESSGREATER }
+  | "<><" { LESSGREATERLESS }
```

Comment:
I tried this on the master branch at the time and it didn't work. Will try again.